Fixed necessary errors

yilmazalican opened the pull request:
I'm new in python but i've developed an web app with Django framework in my school for software Engineering class. But i'm very excited to work with python and its frameworks. I assume that this project is ARSKOM's project and its a web framework written in python. I'm using MacOS El Cap. Got some errors when prepare my environment for this app. First i can't install the requirements. An error was thrown in Terminal about SSL cert verification. But i installed them in by manually looking in the setup.py in root dir. Then i prepared a requirements.txt file for pip package manager. Then i got some errors about dateformat in console of chrome and didn't got the time correctly in TimePicker. Then i fixed it. I know maybe i'm not an expert or well skilled engineer candidate but i'm sure i will develop myself in ARSKOM as fast as i can and i know that i will make huge good differences on company's projects in long term. And i want from you only one chance to evaluate my skillset. Thanks for the invitation for this challenge. Hope we will met soon, Alican.

plq commented:
This is a good enough evaluation of your skill set.

Actually, the test dependencies are handled quite nicely by the setup script.

yilmazalican commented:
I’m using OSX 10.11.5, When i try to apply instructions in readme.md, the first error about SSL certificates. When the setup.py script try to download dependencies from pypi.python.org, it throwed error saying “SSL: CERTIFICATE_VERIFY_FAILED”. Then i’ve installed the dependencies as in the setup.py with pip manually. Then i’ve realized another error in make command. The make command didn’t downloaded the jquery-ui libraries successfully, server responsed as 404. Then i’ve modified the code in the make file then i’ve successfully installed the jquery-ui to the assets folder with make command. As you say i didn’t understand what problem is about test cases, but i’ve seen that there is an error about datetime string in the result of test cases in html format. They are foldered in root dir as test_html. I was not getting the date and time correctly. Then i’ve fixed that so the html output was good to displaying the format and datetime as correct. There is no error anymore. Can you please give me a chance to work with you? I know i’m not good enough but you can try me. I do not expect any income like salary, food, service etc. Anyway thank you for assisting me.

plq commented:
this sounds mac-specific. I don't have a mac so can't suggest anything.
